woocommerce/woocommerce
# Skip payment welcome screen for WCPay banner experiment

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33055 .

This PR updates the WCPay setup button to redirect the users directly to the WC Pay connect page.

original issue 
- https://github.com/woocommerce/woocommerce/issues/32414


### How to test the changes in this Pull Request:

1. Follow test instructions in pbIJXs-1P0-p2
2. Go to `WooCommerce > Settings > Payments` tab.
3. Click WCPay "setup" button
4. Confirm the button redirects you to the WC Pay connect page.

### Other information:




### FOR PR REVIEWER ONLY:
